### PR TITLE
Documentations for Issue No. #23 are added

### PR DESCRIPTION
**Pull request by Vedant-Jayesh-Oza**
# Description

Hey, @ashish-ad the Issue number #23 which was assigned to me is now solved. I have modified the main readme file of the repository and added 2 more readme files named CONTRIBUTION.md and CODE_OF_CONDUCT.md and I have linked these 2 readme files with the main readme file of the repository. My branch document-for-repo is now completely ready to get merged into the main branch.

## Type of change
- Change in the Documentation of the repository 

## Please include a screenshot in case of frontend/UI changes

# Checklist:

- [x] My code follows the style guidelines of this project

### CODE REVIEW

**Review by ashish-ad — COMMENTED** [review]
Hey @Vedant-Jayesh-Oza, is the link shared over here for Figma is the viewable or editable link.

**Comment by Vedant-Jayesh-Oza** [comment]
> Hey @Vedant-Jayesh-Oza, is the link shared over here for Figma is the viewable or editable link.

I have shared the link which I got through mail, it's viewable I guess.

File: README.md [hunk 1]
```diff
@@ -1,52 +1,48 @@
...
-<div align="center" margin-bottom="10em"><a href="https://hacktoberfest.digitalocean.com/"><img src="https://hacktoberfest.digitalocean.com/_nuxt/img/logo-hacktoberfest-full.f42e3b1.svg" alt="Banner" width="60%"/></a></div>
-<br />
-<br />
-The official website of GDSC LBCE community.
+<div align="center" margin-bottom="10em"><a href="https://gdsc.community.dev/dr-lankapalli-bullayya-college-of-engineering-visakhapatnam/"><img src="https://user-images.githubusercontent.com/75005433/145153403-4472e4df-88cf-41c5-8631-97ab86a40267.png" alt="Banner" width="60%"/></a></div>
+
+
+<h2 align="center">About the Repository</h2>
+The Repository belongs to Google Developer Students Club of Dr. Lankapalli Bullayya College of Engineering.
+This is a newly found GDSC club and we are looking for some contributors to help us build our community website from scratch. We always encourage new ideas. Feel free to get in touch with us and join our [discord server](https://discord.gg/ATvz2fqw) for updates.
```
Comment (ashish-ad) [inline README.md:13]: @Vedant-Jayesh-Oza, u need to edit the discord server link it is not showing properly.

**Comment by ashish-ad** [comment]
@Vedant-Jayesh-Oza, may we can use this link for figma <iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="800" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FUg4bm8mjcNafExTiS5TBnr%2FGDSC-LBCE%3Fnode-id%3D0%253A1" allowfullscreen></iframe>

**Comment by ashish-ad** [comment]
@Vedant-Jayesh-Oza, revert the changes and commit it again using this link the iframe tag link is not working 

https://www.figma.com/file/Ug4bm8mjcNafExTiS5TBnr/GDSC-LBCE?node-id=0%3A1

**Comment by Vedant-Jayesh-Oza** [comment]
Yeah will do that

**Review by ashish-ad — APPROVED** [review]
good to go!

**Pull request merged**